pkg/util/log/log.go

```diff
@@ -58,25 +71,16 @@ func newBasicLogger(format dslog.Format, buffered bool) log.Logger {
 			dslog.WithPrellocatedBuffer(logBufferSize),
 		)
 
-		writer = bufferedLogger
-	} else {
-		writer = log.NewSyncWriter(writer)
+		return bufferedLogger
 	}
-
-	if format.String() == "json" {
-		logger = log.NewJSONLogger(writer)
-	} else {
-		logger = log.NewLogfmtLogger(writer)
-	}
-
-	// return a Logger without filter or caller information, shouldn't use directly
-	return log.With(logger, "ts", log.DefaultTimestampUTC)
+	return log.NewSyncWriter(writer)
 }
 
 // NewDefaultLogger creates a new gokit logger with the configured level and format
-func NewDefaultLogger(l dslog.Level, format dslog.Format) log.Logger {
-	logger := newBasicLogger(format, false)
-	return level.NewFilter(log.With(logger, "ts", log.DefaultTimestampUTC), l.Gokit)
+func NewDefaultLogger(lvl dslog.Level, format string) log.Logger {
```

Comment:
I think we can get rid of this function, and just use `InitLogger()` if you follow my suggestion about removing `server.Config` parameter from `InitLogger()`.